### DISCUSSION TITLE
Problem installing the addon (No Module Named 'asset_browser_utilities')

### DISCUSSION THREAD

**Discussion by Amer004** [Gorgious56/asset_browser_utilities#122] [Q&A] [answered]
Hey,
 I am trying to install the addon to export thumbnails for a large Blender file I have, containing about a thousand objects. I tried installing the addon on Blender 3.22 and 3.5 but I am met with an error message saying "No Module Named 'asset_browser_utilities' ".
![image](https://github.com/Gorgious56/asset_browser_utilities/assets/111673885/0d04be04-3df5-470a-9a2a-6c4fc8476a75)

Please tell me what I am doing wrong, or if the addon itself isn't compatible with certain versions of Blender.

Thanks in advance.

**Comment by Gorgious56** [comment, marked as answer]
Hello ! :)

That's a known issue with how github renames the files. See 

https://github.com/Gorgious56/asset_browser_utilities/issues/35

**Reply by Amer004** [reply]
Thanks a lot
